index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capstone Module1</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="about.html">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="icon" href="./Images/logo-no-background.svg" type="image/svg" />
+  </head>
+  <body>
+    <section class="nav-bar">
+      <div id="ham-menu">
+        <img
+          onclick="clickX()"
+          id="x"
+          src="https://img.icons8.com/sf-regular/48/FFFFFF/multiply.png"
+        />
+        <ul class="ham">
+          <li>
+            <a href="#" class="ham-items" onclick="clickX()"
+              >Home</a
+            >
+          </li>
+          <li>
+            <a href="about.html" class="ham-items" onclick="clickX()">About</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Program</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Join</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Sponsor</a>
+          </li>
+        </ul>
+      </div>
+      <header class="head flex-row">
+        <nav>
+          <img
+            onclick="clickMenu()"
+            id="menu"
+            src="./Images/Icon - Menu.png"
+            alt="Menu Logo"
+            class="menu-logo"
+          />
+          <a href="">
+            <img class="logo" src="Images/logo-black.png" alt="logo" />
+          </a>
+          <ul class="menu_items flex-row">
+            <li class="nav_items"><a href="#">Home</a></li>
+            <li class="nav_items"><a href="about.html">About</a></li>
+            <li class="nav_items"><a href="">Program</a></li>
+            <li class="nav_items"><a href="">Join</a></li>
+            <li class="nav_items"><a href="">Sponsor</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div class="main-headline">
+        <h2 class="welcome">Welcome Humans</h2>
+        <h3 class="welcome-2">World is moving with</h3>
```

Comment:
- [ ] please remember you were to follow the provided design guidelines for this project but  personalize the content. your home headline design is kind of different from the original design. i cant see the small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text. like in the original design. remembe the colors and font type also matters
- [ ] also add a background so it can differentiate your headline from your nav bar.
yours
![Screenshot (978)](https://user-images.githubusercontent.com/65298048/233435799-ceb2880f-1152-4fbc-b3bf-b586ae72f3f8.png)
original
![Screenshot (979)](https://user-images.githubusercontent.com/65298048/233435954-e7c86460-4832-4bb9-9e4a-4165b22442bc.png)